wizard/src/components/mapping_column_options.tsx

```diff
@@ -0,0 +1,211 @@
+/**
+ * Copyright 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * Component for showing and updating options for a single column.
+ */
+
+import _ from "lodash";
+import React from "react";
+import { FormGroup, Input, Label } from "reactstrap";
+
+import { MappedThing, MappingType } from "../types";
+import { PlaceDetector } from "../utils/detect_place";
+import { ColumnInfo } from "./mapping_section";
+
+interface MappingColumnOptionsProps {
+  column: ColumnInfo;
+  onColumnUpdated: (column: ColumnInfo) => void;
+  validPlaceTypeProperties: Record<string, Set<string>>;
+  placeDetector: PlaceDetector;
+}
+
+const MAPPED_THING_NAMES = {
+  [MappedThing.UNIT]: "unit of measure",
+  [MappedThing.VALUE]: "observation value",
+};
+
+export function MappingColumnOptions(
```

Comment:
why does this function name start with a capital letter? Also, earlier we have an interface name starting with "Mapping.." and now a function name which also has the same start. Not sure we want to do that? Also, would be great to have a function comment?

Reply:
This is how a React component is defined! And there is a top level comment in this file saying what this component does

Reply:
Cool